services/api-server/src/simcore_service_api_server/models/schemas/model_adapter.py

```diff
@@ -121,3 +125,20 @@ class ServicePricingPlanGetLegacy(BaseModel):
 assert set(ServicePricingPlanGetLegacy.model_fields.keys()) == set(
     _ServicePricingPlanGet.model_fields.keys()
 )
+
+
+class LicensedItemGet(BaseModel):
+    model_config = ConfigDict(
+        populate_by_name=True,
+    )
+    licensed_item_id: LicensedItemID
+    name: str
```

Comment:
what is this name? a display name? a name identifier (i.e. unique, e...)
I would change the field name accordingly in the shcmea